Core/src/main/java/com/plotsquared/core/plot/expiration/ExpireManager.java

```diff
@@ -60,7 +60,6 @@
 
 public class ExpireManager {
 
-    public static ExpireManager IMP;
```

Comment:
```suggestion
/**
* @deprecated Use {@link PlotPlatform#expireManager()} instead
*/
@Deprecated(forRemoval = true, since = "TODO")
public static ExpireManager IMP;
```

This is public API and can't be removed without causing possible regressions for third party consumers.

The import for the class for the javadoc is needed still.